Llk/Sampler/Sampler.php

```diff
@@ -125,6 +131,35 @@ public function getCompiler()
     }
 
     /**
+     * Get a string representation of the current namespace skip token.
+     *
+     * @return  string
+     */
+    protected function getSkipToken()
```

Comment:
I renamed the method to `getSkipTokenAst` (or `getSkipTokenAST` ?) and move the [condition](https://github.com/hoaproject/Compiler/pull/27/files#diff-46b011929b66e4913cb91de7f03537b9R226) before call to `Visitor\Visit::visit()` to avoid visit an empty string instead of a `Token`.